src/payment-account.js

```diff
@@ -13,6 +16,68 @@ class PaymentAccount extends ExchangePaymentAccount {
       return trade;
     });
   }
+
+  sell () {
+    console.log('called sell', this);
+    const sellData = {
+      priceQuoteId: this._quote._id,
+      transferIn: {
+        medium: 'blockchain'
+      },
+      transferOut: {
+        medium: 'bank',
+        mediumReceiveAccountId: this._account.id
+      }
+    };
+    console.log('sell payload', sellData);
+    return this._api.authPOST('trades', sellData);
```

Comment:
The sell method should for the most part just call a class method on `Trade`. See SFOX buy for an example:
https://github.com/blockchain/bitcoin-sfox-client/blob/master/src/payment-account.js#L27-L32

https://github.com/blockchain/bitcoin-exchange-client/blob/master/src/payment-account.js#L19-L35

I think the best approach is to add a sell method to the PaymentAccount exchange class and then call super here, like with SFOX buy.